modules/t/test-genome-DBs/test_collection/core/SQLite/table.sql

```diff
@@ -131,6 +131,22 @@ CREATE TABLE attrib_type (
 
 CREATE UNIQUE INDEX code_idx ON attrib_type (code);
 
+--
+-- Table: biotype
+--
+CREATE TABLE biotype (
+  biotype_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
```

Comment:
I believe a primary key must be not null by definition.

Reply:
this is actually auto generated by the patch_test_databases.pl script...